tests/expectations/compiler/records/same_name_diff_type_fail.out

```diff
@@ -0,0 +1,11 @@
+namespace = "Compile"
+expectation = "Fail"
+outputs = ["""
+Error [ETYC0372119]: Received different types `R` and `R` for the operation `==`.
```

Comment:
For this instance of the error, is it possible to include the program names so that the user understands that the records are fundamentally different?